src/lib/connectors/repos/queries/gauging-stations.js

```diff
@@ -0,0 +1,15 @@
+exports.findGaugingStations = `select 
+water.licence_version_purpose_condition_types.*,
+water.licence_version_purpose_conditions.*,
+water.licence_version_purposes.*,
+water.licence_versions.end_date,
+water.gauging_stations.* from 
+water.gauging_stations join water.gauging_station_condition on (water.gauging_stations.gauging_station_id = water.gauging_station_condition.gauging_station_id)  
+join water.licences on (water.gauging_station_condition.licence_id = water.licences.licence_id)
+join water.licence_versions on (water.licences.licence_id = licence_versions.licence_id)
+join water.licence_version_purposes on (water.licence_versions.licence_version_id = water.licence_version_purposes.licence_version_id)
```

Comment:
I think the join on `licence_version_purposes` should probably be a `left join` since the purpose is optional and might not exist